test/actor.cc

```diff
@@ -49,8 +49,12 @@ TEST(Actor, Basic) {
   auto transit = test::json_to_pt(transit_json);
   ASSERT_NE(transit_json.find(std::to_string(false)), std::string::npos);
 
-  auto status_json = actor.status("");
-  ASSERT_EQ(status_json, "{}");
+  actor.status("");
```

Comment:
Should probably assert that this one is empty?

Reply:
it'll actually return the config and version. but yeah, should be tested anyways